content/next/index.md

```diff
@@ -150,6 +150,10 @@ The first draft of this specification has been written in collaboration with som
 * [unleash](https://github.com/netflix/unleash): a tool for automating the software release and publishing lifecycle.
 * [lerna](https://github.com/lerna/lerna): a tool for managing monorepos, which grew out of the Babel project.
 
+## Tooling for Conventional Commits
+
+* [conform](https://github.com/autonomy/conform): a tool that can be used to enforce conventional commits.
```

Comment:
This tool does more than just enforcing conventional commits. Maybe we should mention that. Reason I say this is https://github.com/marionebl/commitlint can also lint commit messages.

Reply:
👍 Updated.

Reply:
I like the new message better. It still states the feature is to enforce conventional commits but also only briefly described that it can enforce policies. Later on when we add commitlint people won't ask "What's the difference" without clicking on both links.